codec/src/main/java/io/netty/handler/codec/DefaultHeaders.java

```diff
@@ -1170,5 +1170,16 @@ public final V setValue(V value) {
         public final String toString() {
             return key.toString() + '=' + value.toString();
         }
+
+        @Override
+        public boolean equals(Object o) {
```

Comment:
@ryu1-sakai I think we should also override `hashCode` if we override `equals`.

Reply:
@normanmaurer thank you for reminding me. Added `hashCode`.